docs/site/migration/auth/example.md

```diff
@@ -0,0 +1,570 @@
+This example is migrated from
+[loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control),
+and uses the authentication and authorization system in LoopBack 4 to implement
+the access control.
+
+# App Scenario
+
+This example is a donation system. Users can view or make donations to projects
+based on their roles. A project is owned by a user, and a user can create a team
+to involve other users as team members. The system has an administration.
+
+Here is the diagram that describes the models:
+
+![models](../../imgs/tutorials/access-control-migration/auth_example_scenario_model.png)
+
+And the Project endpoints with their allowed roles:
+
+- /Projects/list-projects
+  - everyone
+- /Projects/view-all
+  - admin
+- /Projects/{id}/show-balance
+  - owner, teamMember
+- /Projects/{id}/donate
+  - admin, owner, teamMember
+- /Projects/{id}/withdraw
+  - owner
+
+The diagram for models also include the pre-created data in this application.
+
+For example, u1 owns project1, it also creates a team with u1 and u2 as members.
+Which means u1 is the **owner** of project1, and u1 and u2 are the **team
+members** of project1. And u3 is the **admin**.
```

Comment:
`Which means u1 is the **owner** of project1, and u1 and u2 are the **team members** of project1. And u3 is the **admin**.` -> `This means u1 is the **owner** of project1, u1 and u2 are the **team members** of project1, and u3 is the **admin**.`